src/documentation/pages/components/tabs/examples/_vertical-subtabs-lumen-centurylink.vue

```diff
@@ -0,0 +1,117 @@
+<template lang="pug">
+  <ComponentExample title="Vertical Subtabs" :id="exampleId" additionalClasses="-bg--grey-20" padding="-p--0" :tabs="exampleTabs" :headTabs="headTabs" @chiHeadTabsChange="e => changeSelectedTab(e)">
+    .-p--3(slot="example")
+      div(:class="`-bg--${selectedTabId === 'base' ? 'white' : 'black'} -p--3`")
```

Comment:
Remember that you can use the classes to create a `div` instead of using `div(class: '...')`
```suggestion
      .-p--3(:class="`-bg--${selectedTabId === 'base' ? 'white' : 'black'}` slot="example")
```

Check this in every example.